### ISSUE TITLE
[Improvement][server] Can theHttpTask component support connection timeout and read timeout configuration items

### ISSUE THREAD

**Issue by miki-hmt** [apache/dolphinscheduler#6080]
**Describe the question**
At present, the default read timeout of httptask component is 1 minute. For some APIs that take a long time, an error - read timeout error will be directly reported. Can the dolphin scheduler support the UI layer to configure the read timeout parameter later？
```
/**
     * request config
     * @return RequestConfig
     */
    private RequestConfig requestConfig() {
        return RequestConfig.custom().setSocketTimeout(MAX_CONNECTION_MILLISECONDS).setConnectTimeout(MAX_CONNECTION_MILLISECONDS).build();
    }
```

**What are the current deficiencies and the benefits of improvement**
- After the read timeout UI configuration is supported, APIs that take a long time to read and download some files can be used normally. Otherwise, the httptask component function of the current system will be relatively weak
- At present, the read timeout is one minute by default. If the timeout alarm function of httptask is set to more than one minute, it is useless, because the read timeout error will trigger the task to stop before the timeout alarm

**Which version of DolphinScheduler:**
 -[1.3.7-release]

**Describe alternatives you've considered**
Look forward to the reply from the big guys！

**Comment by ruanwenjun** [comment]
What do you think if this parameter can be configured ?

**Comment by zhuangchong** [comment]
@miki-hmt 

The problem you describe has been resolved in 1.3.7 branch.
![image](https://user-images.githubusercontent.com/37063904/131809771-d2e12a07-1b29-482f-bbbc-2404029a25a7.png)

**Comment by miki-hmt** [comment]
haha, Because I currently use the 1.36 branch, I don't see any changes. Thanks a lot
